src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.ts

```diff
@@ -55,6 +57,31 @@ export class DashboardComponent implements OnInit, OnDestroy {
   healthData: any;
   categoryPgAmount: Record<string, number> = {};
   totalPgs = 0;
+  queriesResults: any = {
+    USEDCAPACITY: '',
+    IPS: '',
+    OPS: '',
+    READLATENCY: '',
+    WRITELATENCY: '',
+    READCLIENTTHROUGHPUT: '',
+    WRITECLIENTTHROUGHPUT: '',
+    RECOVERYTHROUGHPUT: ''
+  };
+  usedCapacity: any;
+  ips: any;
+  ops: any;
+  readLatency: any;
+  writeLatency: any;
+  readClientThroughput: any;
+  writeClientThroughput: any;
+  recoveryThroughput: any;
```

Comment:
There are too many any types here. Output of prometheus queries has a clear format we can classify:
```
{"resultType": "matrix", "result": [{"metric": {}, "values": [[1673362508, "0"], [1673362538, "0"], [1673362568, "0"], [1673362598, "0"], [1673362628, "0"], [1673362658, "0"]]}]}
values == Array<DataPoint>
Datapoint {
  timestamps: number;
  value: string
}